modules/navier_stokes/doc/content/modules/navier_stokes/index.md

```diff
@@ -69,4 +69,31 @@ future these kernel sets will be consolidated into a single set.
+
+!table id=navier_stokes_solver_summary caption=Summary of Navier-Stokes Solver Capabilities
+| Capability         | model                     | FE Newton  | FV Newton                                                    | FV Nonlinear SIMPLE                                          | FV Linear SIMPLE       |
+| ------------------ | ------------------------- | ---------- | -----------------------------------------------------------  | ------------------------------------------------------------ | ---------------------- |
+| Transient          |       --                  | Yes        | Yes                                                          | Yes                                                           | Yes                                                          |
+| Turbulence         | Mixing length             | Yes        | Yes                                                          | Yes                                                          |                        |
+|                    | $k-\epsilon$              |            | Yes                                                          | Yes                                                          | under development      |
+|                    | $k-\omega$ SST            |            |                                                              | in [PR #28151](https://github.com/idaholab/moose/pull/28151) |                        |
+| Two-phase          | Mixture model             | Yes        | Yes                                                          | Yes                                                          | in [PR #29614](https://github.com/idaholab/moose/pull/29614) |
+|                    | Eulerian-Eulerian         |            |                                                              | Yes                                                          |                        |
+| Porous Flow        |       --                  | Yes        | Yes                                                          | Yes                                                          |                        |
+| Compressibility    | Incompressible            | Yes        | Yes                                                          | Yes                                                          | Yes                    |
+|                    | Weakly compressible       |            | Yes                                                          | Yes                                                          |                        |
+|                    | Compressible              |            |                                                              |                                                              | Yes                    |
```

Comment:
I think Compressible is only FV Newton.

Reply:
let's squash it in please. 5 commits for a single table would be excessive imo